### PR TITLE
fix: Update comments in index files

### PR DESCRIPTION
**Pull request by xil222**
Update the comment in index.ts for VerifyIdToken and VerifySessionCookie be same as in auth.ts.

### CODE REVIEW

**Review by egilmorez — APPROVED** [review]
Thanks!

File: src/auth/index.ts [hunk 1]
```diff
@@ -1707,11 +1707,15 @@ export namespace auth {
     updateUser(uid: string, properties: UpdateRequest): Promise<UserRecord>;
 
     /**
-     * Verifies a Firebase ID token (JWT). If the token is valid, the promise is
-     * fulfilled with the token's decoded claims; otherwise, the promise is
-     * rejected.
-     * An optional flag can be passed to additionally check whether the ID token
-     * was revoked.
+     * Verifies a JWT auth token. Returns a promise with the tokens claims.
```
Comment (hiranya911) [inline src/auth/index.ts:1710]: token's claims.

File: src/auth/index.ts [hunk 2]
```diff
@@ -1707,11 +1707,15 @@ export namespace auth {
...
+     * Rejects the promise if the token cannot be verified.
+     * If `checkRevoked` is set to true, first verifies whether the corresponding
+     * user is disabled.
+     * If yes, an auth/user-disabled error is thrown.
```
Comment (hiranya911) [inline src/auth/index.ts:1714]: Might want to code format the error code. Here and below.

**Review by hiranya911 — APPROVED** [review]
LGTM with a couple of suggestions.

File: src/auth/auth.ts [hunk 3]
```diff
@@ -104,14 +104,14 @@ export class BaseAuth<T extends AbstractAuthRequestHandler> implements BaseAuthI
   }
 
   /**
-   * Verifies a JWT auth token. Returns a promise with the tokens claims.
+   * Verifies a JWT auth token. Returns a promise with the token‘s claims.
    * Rejects the promise if the token cannot be verified.
    * If `checkRevoked` is set to true, first verifies whether the corresponding
    * user is disabled.
-   * If yes, an auth/user-disabled error is thrown.
+   * If yes, an `auth/user-disabled`s error is thrown.
```
Comment (lahirumaramba) [inline src/auth/auth.ts:111]:
```suggestion
   * If yes, an `auth/user-disabled` error is thrown.
```

**Review by lahirumaramba — APPROVED** [review]
LGTM with a single nit :)
Thanks!

**Pull request merged**